AllCoreDevs-Meetings/Meeting 146.md

```diff
@@ -0,0 +1,174 @@
+
+**Trent**
+* We will be having the final, merged community call on the 9th Sept. So, obviously you want to prepare for Bellatrix before that, but, there will be one more session. 
+
+**Ansgar Dietrichs**
+* That's good to hear. Do we, someone just, remind the listeners when exactly Bellatrix is expected to happen. 
+
+**Trent**
+* I believe it's September 6th, but I will double check on that. 
+
+**Ansgar Dietrichs**
+* Yeah. I think that, that seems what I remember as well. And of course I was Bellatrix. This, not the, not the proof work and predictability of the timing. So, yeah, I think, I think it is. So again, for anyone listening, you know, you should really have your setup ready by then because that's when the consensus layer forks. Okay. I might, I see you have your hand raised. 
+
+**Ahmad Bitar**
+* Yeah. So just need to encourage people to upgrade their clients because only right now, 53% of clients are merchant ready. So we need more people to update their clients, be ready for them. 
+
+**Micah Zoltu**
+* Does that 50% to 53% of staker or 53% of users are not operators. 
+
+**Ahmad Bitar**
+* I'm getting my data from ethn.org, not entirely sure. I'm pretty sure it's just node operators. 
+
+**Ansgar Dietrichs**
+* Okay. Yeah. That's, that's useful context, of course. for non-statin auto operators, there's always less incentive to make sure that they're fully, fully set up. So we, so we might expect that this will not quite reach a hundred percent before the merger, but if you are not a blatant, you want to continue following the chain. then, you of course, have dropped bit. So please do that. Any other much updates? I mean, it sounds like we're just not right on track. quite exciting. well then, let's just briefly go to the other general point, just other, do we, do we have anything, anything non-matriculated, we can become discuss, right now. Okay. Sounds like everyone is focused on the image as it should be. great. So then with that, this might be one of the shortest allcore dev calls in history. happy coincidence because, you know, I couldn't couldn't really break anything. It's a great like idea of anything in the, And they'll talk about it. 
```

Comment:
```suggestion
* Okay. Yeah. That's, that's useful context, of course. for non-statin auto operators, there's always less incentive to make sure that they're fully, fully set up. So we, so we might expect that this will not quite reach a hundred percent before the merge, but if you are not a blatant, you want to continue following the chain. then, you of course, have dropped bit. So please do that. Any other much updates? I mean, it sounds like we're just not right on track. quite exciting. well then, let's just briefly go to the other general point, just other, do we, do we have anything, anything non-matriculated, we can become discuss, right now. Okay. Sounds like everyone is focused on the image as it should be. great. So then with that, this might be one of the shortest allcore dev calls in history. happy coincidence because, you know, I couldn't couldn't really break anything. It's a great like idea of anything in the, And they'll talk about it. 
```